src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -6030,18 +6030,14 @@ void MusicXmlParserPass2::divisions()
 // the type for all rests.
 // Sibelius calls all whole-measure rests "whole", even if the duration != 4/4
 
-static bool isWholeMeasureRest(const String& type, const Fraction dura, const Fraction mDura)
+static bool isWholeMeasureRest(const String& type, const Fraction restDuration, const Fraction measureDuration)
 {
-    if (!dura.isValid()) {
-        return false;
-    }
-
-    if (!mDura.isValid()) {
+    if (!restDuration.isValid() || !measureDuration.isValid()) {
         return false;
     }
 
-    return (type.empty() && dura == mDura)
-           || (type == u"whole" && dura == mDura);
+    return (type.empty() && restDuration == measureDuration)
+           || (type == u"whole" && restDuration == measureDuration && restDuration != Fraction(1, 1));
```

Comment:
```suggestion
           || (type == u"whole" && restDuration == measureDuration);
```
The unit tests are the desired outcome. This condition is causing the test failures - looks like it sneaked back in after the rebase.